test/config_test.exs

```diff
@@ -14,7 +14,7 @@ defmodule ConfigTest do
                         dev: %Environment{profile: %Profile{dev_mode: true, include_erts: false}},
                         prod: %Environment{profile: %Profile{dev_mode: false, include_erts: true, strip_debug_info: false}}},
                       releases: %{
-                        standard_app: %Release{version: "0.0.1", applications: [:elixir, :iex, :sasl, :standard_app]}},
+                        standard_app: %Release{version: "0.0.1", applications: [:elixir, :iex, :sasl]}},
```

Comment:
It gets added later in `Utils.get_apps/1`, so it's fine.
I'd like to get the rest of "default" applications moved from `config.ex` there as well to keep the entire application resolution thing together, but that's out of the scope of this change.